.github/workflows/run-integration-tests.yml

```diff
@@ -12,8 +12,16 @@ concurrency:
 
 jobs:
   all-integration-tests:
-    name: 'Run integration tests on all environments'
+    name: 'Run integration tests on all Aurora environments'
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I'd prefer to let integration suite to deal with all different environments. I see no point to have matrix here.

Reply:
This is a change that @karenc-bq suggested. One advantage is that the tests are ~20 minutes faster because mysql/pg are tested in parallel. Not sure if she had other advantages in mind

Reply:
Mostly to speed up the tests. MySQL tests are slower now so i think we actually save half an hour